openassessment/xblock/submission_mixin.py

```diff
@@ -783,11 +785,23 @@ def submission_path_and_context(self):
             context['save_status'] = self.save_status
             context['enable_delete_files'] = True
 
+            submit_enabled = True
```

Comment:
You might ask why I moved this, my answer is, why the heck did we add in the teams block in the middle of the function?????

Reply:
literally LOL